changelog/5039.bugfix.rst

```diff
@@ -0,0 +1 @@
+Fix regression with ``--pdbcls``, which stopped working with local modules in 4.0.0.
```

Comment:
Btw, you think this regression deserves a quick fix release @blueyed? If so please feel free to prepare one (from master of course). 👍

Reply:
No, I don't think it is urgent - after all it might help to discover the new feature behind it.. ;)